doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -333,9 +327,9 @@ For a better outline on what we DO want to do, see https://github.com/CleverRave
 
 From a game balance perspective zombies are very abundant and should only be used under limited lore-friendly circumstances in consumables, trivial early items or with lasting risks and drawbacks.
 
-#### Conversations with monsters and other things: yes, planned for 0.G
+#### Conversations with monsters and other things: Partially impelmented
 
-There is an extensive and sophisticated system for talking with NPCs, but you can't have a conversation with your genetically modified, super-intelligent dog, nor with your intelligent sword from Magiclysm, nor with your car.
+This is partially implemented already with monsters, items, furniture, as the player character can have conversations with them via NPC-style dialogue. However, support for dialogue functions is limited; you can't trade with monsters, and a significant quantity of edge cases for oddball functions remain unsupported for non-NPC talkers. Conversations with vehicles are entirely unsupported, meaning you can't talk with your car or some super-intelligent AI inside it.
```

Comment:
I've seen a mod somewhere in the hinterlands with trading monsters.  Does it not work? <https://github.com/puninokp/Magiclysm_with_Random_Dungeons/blob/master/monsters/ally_monsters.json>

Reply:
The mod doesn't have the player trading directly with monsters, but an NPC.

Reply:
Not quite what I mean with the new text in this PR.

Reply:
Because it doesn't open a trade window, you can only "trade" for specific items.  I see waht you are saying.